### ISSUE TITLE
Persian Alphabets Classification using NLP

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#415]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Persian Alphabets Classification using NLP</br>
:red_circle: **Aim** : The aim of this project is to classify the persian alphabets using NLP methods.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/mostafamohammadi1/persian-alphabets-and-numbers</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Sgvkamalakar** [comment]
- Full name: S G V Kamalakar

- GitHub Profile Link: https://github.com/Sgvkamalakar

- Email ID: sgvkamalakar@gmail.com

- Participant ID (if applicable):

- Approach for this Project: The approach involves utilizing a **Convolutional Neural Network (CNN), transform-learning with pre-trained models, and utilizing the power of transformers** to accomplish Persian alphabet classification.

- What is your participant role? (Mention the Open Source program): **SWOC S4**

I'll submit a pull request, following the guidelines within the next few days. Would u like to assign the issue #415 to me?

**Comment by abhisheks008** [comment]
Use at least 2-3 deep learning methods for this project, to find out the best fitted model based on the accuracy scores for this dataset.

Issue assigned to you @Sgvkamalakar. You can start working on it.
